doc/integrating/integration.md

```diff
@@ -139,6 +139,7 @@ const locale = {
     code: "en_US",
     thousandsSeparator: ",",
     decimalSeparator: ".",
+    weekStart: 7,
```

Comment:
the 1-7 mapping could be explained in the doc as well

Reply:
Also maybe add that information in the Localisation sidepanel?

Reply:
Updated :)